connectors/sources/sharepoint_online.py

```diff
@@ -1462,7 +1466,6 @@ async def site_list_items(
             list_item_natural_id = list_item["id"]
             list_item["_id"] = f"{site_list_id}-{list_item['id']}"
             list_item["object_type"] = "list_item"
-            list_item["_original_filename"] = list_item.get("FileName", "")
```

Comment:
I was getting errors that ListItems could not select `FileName` and that this wasn't a valid field. So I deleted this usage, since it would always lead to a `""` value

Reply:
Nice catch!